src/app/notebooks-warning/notebooks-warning.component.html

```diff
@@ -0,0 +1,5 @@
+<mat-card class="alert-info mat-elevation-z w-100" data-cy="notebooks-warning">
+  <mat-icon>warning</mat-icon>
+  <strong> Notice:</strong> This is an early access feature. Consider this a prototype that may be more likely to have breaking changes
```

Comment:
The text for this is very generic, not specific to notebooks. That means it could be used for other new features in the future that we first roll out in preview mode.

Which is a long-winded way of saying that you could rename this component to something more generic like PreviewWarningComponent.